portfolio/src/main/webapp/style.css

```diff
@@ -40,13 +67,28 @@ footer {
   border-radius: 10px;
 }
 
-.buttonBlue {background-color: #0e76a8; color:white;}
-.buttonGray {background-color: #e8e8e8; color:black;}
+.buttonBlue {
+  background-color: #0e76a8; 
+  color:white;
+}
+
+.buttonGray {
+  background-color: #e8e8e8; 
+  color:black;
+}
+
+/*Images - Profile and Non-Profile*/
+.profile {
+  padding: 5px 13px 5px 13px;
+  margin: 10px;
+  border: 2px outset; 
+  float: right;
+}
 
-img {
-  padding: 5px;
-  margin: 15px;
-  border: 2px outset;
+.img-nonprofile {
+  padding: 0px;
+  margin: 0px;
+  border: 0px; 
```

Comment:
I guess 0 should be the default value for the CSS class properties, try to remove it and see if it stays the same

Reply:
Thanks! I removed it and yes, it stays the same.